templates/components/listings/groups/entry.html.twig

```diff
@@ -7,6 +7,7 @@
         </h2>
         {% block group_description %}
             {{ group.description|purify }}
+            {% if group.closed %}<p><em>Group closed on {{ w3c_date_format(group.closedDate)|raw }}</em></p>{% endif %}
```

Comment:
It seems important to indicate that the group is closed, so I woudl suggest putting this before the group's description.

```suggestion
            {% if group.closed %}<p><em>Group closed on {{ w3c_date_format(group.closedDate)|raw }}</em></p>{% endif 
            {{ group.description|purify }}
%}
```